src/convertToObject.js

```diff
@@ -14,14 +14,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  const arr = sourceString.split(';').map(el => el.trim())
-    .filter(element => element.length > 0);
-  const array = arr.map(el => el.split(':'));
-  const array1 = array.map(el => el.map(x => x.trim()));
   const object = {};
```

Comment:
You can change the variable name `from` object to `cssStyled`; it looks better.

Reply:
agreed.